fake_switches/juniper/juniper_netconf_datastore.py

```diff
@@ -133,8 +133,8 @@ def interface_to_etree(self, port):
         if port.description is not None:
             interface_data.append({"description": port.description})
 
-        if port.shutdown is not None:
-            interface_data.append({("disable" if port.shutdown else "enable"): ""})
+        if port.shutdown is not None and port.shutdown:
```

Comment:
![capture d ecran 2016-03-01 a 09 55 10](https://cloud.githubusercontent.com/assets/8175434/13430611/c9309458-df93-11e5-9599-7b70f2b404ce.png)

I don't think so. Shutdown is a boolean.